data/menu.xml.ui

```diff
@@ -56,14 +53,18 @@
         <property name="margin">10</property>
         <child>
           <object class="GtkModelButton">
-            <property name="visible">True</property>
+            <property name="height_request">30</property>
+            <property name="visible">True</property>            
+            <property name="text" translatable="yes">Tabs position</property>
+            <property name="menu_name">menu</property>
             <property name="inverted">True</property>
             <property name="centered">True</property>
-            <!-- <property name="action-name">app.tabsPosition</property> -->
-            <!-- <property name="action-target">"top"</property> -->
-            <property name="text" translatable="yes">Tabs position</property>
-            <property name="menu-name">main</property>
           </object>
+          <packing>
+            <property name="expand">False</property>
+            <property name="fill">True</property>
+            <property name="position">0</property>
+          </packing>
```

Comment:
Could you provide a screenshot before/after to make reviewing easier? Looks good though